integrations/stripe/src/setup/channels.ts

```diff
@@ -0,0 +1,47 @@
+import type { Channels } from '../misc/types'
+
+class NotImplementedError extends Error {
+  constructor() {
+    super('Not implemented')
+  }
+}
+
+export const channels: Channels = {
+  channel: {
```

Comment:
**chore:** when you remove the channel `channel` from definitions, change this to an empty object